setup.sh

```diff
@@ -300,9 +300,10 @@ function synchronize_files()
   if command_exists "zsh"; then
     # Add tabcompletion to zshrc
     if [[ -f "$HOME/.zshrc" || -L "$HOME/.zshrc" ]]; then
-      echo "# Enable bash completion for zsh" >> "$HOME/.zshrc"
-      echo "autoload bashcompinit && bashcompinit" >> "$HOME/.zshrc"
+      safe_append '# Enable bash completion for zsh' "$HOME/.zshrc"
```

Comment:
couldnt this 2 lines be merged into one?

Reply:
Apparently yes, I was afraid that the text would be split into 2 arguments and that would mess up the function, but I changed it and it seems to work.